apps/api-reference/next.config.js

```diff
@@ -9,6 +9,18 @@ export default {
     },
   },
 
+  redirects: async () => [
+    {
+      source: "/home",
+      destination: "/",
+      permanent: true,
+    },
```

Comment:
I'm not sure I understand why this redirect is here, `/home` doesn't really mean anything?  I'm assuming this has to do with redirecting the old docs pages URLs and if that's the case, it's better to add the redirect to the docs app, where the URL used to be, rather than adding it here.